core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -431,6 +458,11 @@ public final boolean containsObject(IN_NAME name, Object value) {
         return contains(name, fromObject(value));
     }
 
+    @Override
+    public final boolean containsBoolean(IN_NAME name, boolean value) {
+        return contains(name, String.valueOf(value));
```

Comment:
This seems incorrect? Should match `1` and `0` as well as `true` and `false`, right?

Reply:
But then we have to add a method like
`containsBoolean(IN_NAME name, String value) {...}`
Do we need that? 🤔

Reply:
Let me make it case insensitive by the way. 😄

Reply:
No. You can do `getBoolean(name)` and then compare.

Reply:
Thanks! Now I get it. 😓

Reply:
> No. You can do getBoolean(name) and then compare.

Hmm If we use `getBoolean` inside of `containsBoolean`, it checks only first entry using `get(name)`?
Maybe `containsBoolean` should check all entries like other `contains*` 🤔

Please check my Fix PR: https://github.com/line/armeria/pull/5319 for details 🙇